bom/pom.xml

```diff
@@ -1062,29 +1067,29 @@
             </dependency>
             <!-- MicroStream -->
             <dependency>
-            	<groupId>io.helidon.integrations.microstream</groupId>
-            	<artifactId>helidon-integrations-microstream</artifactId>
-            	<version>${helidon.version}</version>
+              <groupId>io.helidon.integrations.microstream</groupId>
```

Comment:
Hmm; not sure what these changes are; will investigate.

Reply:
Aha, whatever commit introduced the MicroStream stuff used tabs (!). My editor changed them to two spaces. I guess they should be four spaces. Will update.